controllers/humiocluster_controller.go

```diff
@@ -2274,6 +2345,12 @@ func (r *HumioClusterReconciler) cleanupUnusedResources(ctx context.Context, hc
 				withMessage(err.Error()))
 		}
 	}
+
+	if err := r.cleanupOrphanedPDBs(ctx, hc, &humioNodePools); err != nil {
+		return r.updateStatus(ctx, r.Client.Status(), hc, statusOptions().
+			withMessage(err.Error()))
+	}
+
```

Comment:
The same comment goes here as https://github.com/humio/humio-operator/pull/900/files#r1930459818
Cleanup of PDB objects should already be garbage collected by k8s out of the box as long as owner references is properly set up.

Reply:
I wanted to be extra careful here. Deleted the conditional statement above.